pkg/azdiskschedulerextender/azdiskschedulerextender_metrics.go

```diff
@@ -84,6 +84,7 @@ func RegisterMetrics(schedulerMetrics ...metrics.Registerable) {
 
 func DurationSince(functionName string, start time.Time) float64 {
 	requestDuration := time.Since(start).Seconds()
-	klog.V(2).Infof("Call to %v took: %f ms\n", functionName, requestDuration/1000)
+	keysAndValues := []interface{}{"latency_seconds", requestDuration, "request", functionName}
+	klog.V(2).InfoS("Observed Request Latency", keysAndValues...)
```

Comment:
You can also pass the key values directly instead of constructing an array first, e.g.
```go
klog.V(2).InfoS("Observed Request Latency", "latency_seconds", requestDuration, "request", functionName)
```

Reply:
Yeah, I know but I thought this made the grouping clearer and the lines shorter. Should I update it?

Reply:
I don't have a strong opinion one way or the other. Line length can be mitigated by breaking the statement across multiple lines.